### ISSUE TITLE
support minizip-ng 4.0

### ISSUE THREAD

**Issue by MehdiChinoune** [AcademySoftwareFoundation/OpenColorIO#1800]
The new version has broken API.

**Comment by Future998** [comment]
Yes, the build of the latest version of opencolorio (2.2.1) with the new version of minizip-ng (4.0.0) broke.

**Comment by Pro-pra** [comment]
Yes minizip-ng say:
```
Functions ending in _create( will no longer accept a first parameter. This forces a single way to get a pointer to new object and is more likely to result in a check of its value.
```

**Comment by brad0** [comment]
https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1806

**Comment by brad0** [comment]
This has been fixed with bdc4cd124140f997cdec1c5d7db72b1550fe7eac
